modules/api-management/service/apis/main.bicep

```
@@ -15,8 +15,10 @@ param apiRevisionDescription string = ''
 
 @description('Optional. Type of API to create. * http creates a SOAP to REST API * soap creates a SOAP pass-through API.')
```

Description would need an update to
```suggestion
@description('Optional. Type of API to create. * http creates a REST API * soap creates a SOAP pass-through API * websocket creates websocket API * graphql creates GraphQL API.')
```